front_end/html/edit_assignment.html

```diff
@@ -45,7 +45,7 @@ <h2>Create Assignment</h2>
           <a href="/delete_assignment/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}">Delete assignment</a> |
           <a href="/delete_assignment_submissions/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}">Delete assignment submissions</a>
       {% end %}
-  </p>
+  </p></br>
```

Comment:
There's actually no such thing as a closing br tag. But you can have a self-closing tag`<br />`. Perhaps that's what you meant. Please play with it a bit more to see what will work. Please also check on two different browsers to make sure it looks right on both.